pkg/microservice/picket/client/policy/rolebinding.go

```diff
@@ -25,3 +25,11 @@ func (c *Client) ListRoleBindings(header http.Header, qs url.Values) ([]*RoleBin
 
 	return res, nil
 }
+
+func (c *Client) DeleteRoleBindings(userID string,header http.Header, qs url.Values) ([]byte,error) {
+	url := "rolebindings/bulk-delete"
+
+	qs.Add("userID", userID)
+	res, err := c.Post(url, httpclient.SetHeadersFromHTTPHeader(header), httpclient.SetQueryParamsFromValues(qs),httpclient.SetBody([]byte("{}")))
+	return res.Body(),err
```

Comment:
Will caller of this method close response body?

Reply:
https://github.com/go-resty/resty/issues/151#issuecomment-387501756